### PR TITLE
[pt-br]: add sidebar to all Glossary pages

### PR DESCRIPTION
**Pull request by leon-win**
### Description

Add sidebar (with list of terms) to all Glossary pages in `pt-BR` locale.

### Motivation

Sidebar is useful for navigation in Glossary, so this change will improve user experience.

### Related issues and pull requests

Fixes #16942
Relates to #16863, #16943, #16970, #16979

### CODE REVIEW

File: files/pt-br/glossary/index.md [hunk 1]
```diff
@@ -1,16 +1,15 @@
 ---
 title: "Glossário do MDN Web Docs: Definições de termos relacionados à Web"
+short-title: Glossário do MDN Web Docs
 slug: Glossary
 ---
 
+{{GlossarySidebar}}
+
+{{GlossarySidebar}}
+
```
Comment (josielrocha) [inline files/pt-br/glossary/index.md:10]: Duplicated content

```suggestion
{{GlossarySidebar}}

```

**Review by josielrocha — COMMENTED** [review]
Hello @leon-win!

Thank you for your help. I made a little suggestion. Could you please fix it?

**Comment by leon-win** [comment]
> Hello @leon-win!
> 
> Thank you for your help. I made a little suggestion. Could you please fix it?

Hello @josielrocha!

Yes, of cource!
That was my fault :sweat: 

Fixed :)

**Comment by josielrocha** [comment]
> > Hello @leon-win!
> > Thank you for your help. I made a little suggestion. Could you please fix it?
> 
> Hello @josielrocha!
> 
> Yes, of cource! That was my fault 😓
> 
> Fixed :)

Thank you @leon-win, everything looks good now.

**Review by josielrocha — APPROVED** [review]

**Pull request merged**